src/components/Search.js

```diff
@@ -10,12 +11,18 @@ const allSuggestions = structure.sections
   .reduce((acc, val) => acc.concat(val), [])
   .sort();
 
-export default () => {
+const SearchButton = ({ setSearchOpen, searchOpen }) => {
```

Comment:
Wondering if we should keep the name `Search` to align with the file name, and because it's a bit more than a `Button`?

Reply:
The issue I was facing here was that I was getting an error that 'Search' is already defined because there is a grommet-icon called Search that I'm importing. I agree that there could be a more encompassing name than `SearchButton` since it is more than the button.